forge.config.js

```diff
@@ -38,6 +38,46 @@ module.exports = {
     },
   ],
   packagerConfig: {
+    afterCopyExtraResources: [
+      (buildPath, electronVersion, platform, arch, callback) => {
+        const path = require('path');
+        const fs = require('fs');
+        const glob = require('glob');
+        console.log('Running afterCopyExtraResources hook...');
+        console.log({ arch, buildPath, electronVersion, platform });
+
+        const chromiumDir = path.join(
+          buildPath,
+          'Loadmill.app',
+          'Contents',
+          'Resources',
+          'standalone_playwright',
+          'node_modules',
+          'playwright-core',
+          '.local-browsers',
+        );
+
+        console.log({ chromiumDir });
+
+        try {
+          const matches = glob.sync(
+            path.join(chromiumDir, '**/gpu_shader_cache.bin'),
+            { nodir: true },
+          );
+
+          matches.forEach(file => {
+            console.log(`Changing permissions for: ${file}`);
+            fs.chmodSync(file, 0o644);
```

Comment:
How is this working on Windows?

Reply:
Like any other NodeJS file system method, it works on all major OS.

```
Running afterCopyExtraResources hook...
{
  arch: 'x64',
  buildPath: 'C:\\Users\\circleci\\AppData\\Local\\Temp\\electron-packager\\tmp-8pyzNm',
  electronVersion: '30.5.1',
  platform: 'win32'
}
{
  chromiumDir: 'C:\\Users\\circleci\\AppData\\Local\\Temp\\electron-packager\\tmp-8pyzNm\\Loadmill.app\\Contents\\Resources\\standalone_playwright\\node_modules\\playwright-core\\.local-browsers'
}
Permissions changed successfully.
```

https://app.circleci.com/pipelines/github/loadmill/desktop-app/783/workflows/fbcdc9e3-bc4a-4dcb-8629-fc9b3102c04b/jobs/902/parallel-runs/0/steps/0-109